run/cifs_archive/write-archive-configs-to.sh

```diff
@@ -2,7 +2,15 @@
 
 FILE_PATH="$1"
 
-(
-  echo "username=$SHARE_USER"
-  echo "password=$SHARE_PASSWORD"
-) > "$FILE_PATH"
+if [[ "$DOMAIN_ENABLE" = false ]]; then
```

Comment:
Same here, this should be conditional on the SHARE_DOMAIN variable itself.
The "if" should be of the form:
```
if condition
then
```
to match the style of the rest of the code in the project.
Instead of having two completely separate code blocks, just surround the `echo domain ...` part with the `if`.